File: tuts/001-lightsail-gs/javascript/scenarios/getting-started.js
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

import { LightsailClient } from "@aws-sdk/client-lightsail";
import { randomUUID } from "crypto";

const main = async () => {
  const client = new LightsailClient({});
  const suffix = randomUUID().slice(0, 8);
  console.log("Running lightsail getting started scenario...");
  // TODO: implement setup, interact, teardown
  console.log("Scenario complete.");
};

main();

File: tuts/001-lightsail-gs/go/scenarios/getting_started.go
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

package main

import (
	"context"
	"fmt"
	"github.com/aws/aws-sdk-go-v2/config"
	aws_lightsail "github.com/aws/aws-sdk-go-v2/service/lightsail"
)

func main() {
	cfg, _ := config.LoadDefaultConfig(context.TODO())
	client := aws_lightsail.NewFromConfig(cfg)
	_ = client
	fmt.Println("Running Lightsail getting started scenario...")
	// TODO: setup, interact, teardown
}
